docs/src/details/xml/integration/renesas-dev-tools.rst

```diff
@@ -0,0 +1,45 @@
+.. _editor_integration_renesas:
+
+============================
+Integrations with Renesas Dev Tools
+============================
+
+Eclipse / E2Studio
+******************
+
+Setup
+-----
+
+First, download and install the LVGL Editor from `pro.lvgl.io <https://pro.lvgl.io/download>`_.
+
+LVGL provides an Eclipse / E2Studio plugin to facilitate working with LVGL projects. You can download it from `here <https://github.com/lvgl/lvgl_editor/releases/tag/v1.0.0>`_.
```

Comment:
Let's use the generic release page. The latest release will be on the top anyways.

```suggestion
LVGL provides an Eclipse / E2Studio plugin to facilitate working with LVGL projects. You can download it from `here <https://github.com/lvgl/lvgl_editor/releases>`_.
```